components/requests/ProcessingTasksCard.tsx

```diff
@@ -0,0 +1,143 @@
+import { Divider, VStack, Button, Text, Link } from '@chakra-ui/react'; // Chakra UI
+import PermitHolderInfoCard from '@components/internal/PermitHolderInfoCard'; // Custom Card Component
+import AssignNumberModal from '@components/requests/modals/AssignNumberModal'; // AssignNumber Modal component
+import ProcessingTaskStep from '@components/requests/ProcessingTaskStep'; // Processing Task Step
+import { useState } from 'react'; // React
+
+type ProcessingTasksCardProps = {
+  readonly applicationProcessingStepsCompleted: number[];
```

Comment:
I think this is a bit redundant since the readonly is already present and `number[]` is a valid way to represent it in TS, but I'll make the change regardless

Reply:
Actually, never mind - changing to ReadonlyArray<number> introduces some weird TypeScript errors where you can't use, for instance, `arr.includes(x)` where `x` isn't itself readonly. Gonna leave it as is (`number[]`)